README.md

```diff
@@ -10,7 +10,7 @@ fraction of the codebase.
 There are a few steps you have to carefully consider before building and installing
 OpenDoas:
 
-* There are less eyes on random doas ports, just because `sudo` had a vulnerability
+* There are less eyes on random `doas` ports, just because `sudo` had a vulnerability
```

Comment:
"fewer eyes" ?

Reply:
Verbiage carried forward from previous author.

Reply:
Sure, I can incorporate that suggestion.